configure

```diff
@@ -1775,7 +1895,7 @@ echo "NATIVECCCOMPOPTS=$nativecccompopts" >> Makefile
 echo "NATIVECCPROFOPTS=$nativeccprofopts" >> Makefile
 echo "NATIVECCLINKOPTS=$nativecclinkopts" >> Makefile
 echo "NATIVECCRPATH=$nativeccrpath" >> Makefile
-echo "NATIVECCLIBS=$cclibs $dllib" >> Makefile
+echo "NATIVECCLIBS=$cclibs $nativecclibs $dllib" >> Makefile
```

Comment:
Is this change needed for spacetime? I'm slightly worried it might break something else.